rest-api/src/test/java/org/modelix/samples/some/RestApiTest.java

```diff
@@ -14,9 +14,9 @@ class RestApiTest {
 	EmbeddedApplication<?> application;
 
 	// TODO: broke with version 2.0
```

Comment:
```suggestion
```

Reply:
this project will be removed anyways in the future :)